docs/getting-started/quickstart/index.md

```diff
@@ -24,12 +24,16 @@ The quickstart guide lets you make a series of selections to put together your c
 3. Studio setup
 4. Deployment
 
+![Screenshot of quickstart start step](./Quickstart.png)
+
 ## Starter template
 The starter template defines what front-end and source system your website starts with. We may also import some demo content into your source system. You can change all this later.
 
-If you are not interested in importing demo data into Atama and your source system, you may skip the quickstart altogether. Simply click the "Skip" button in the upper right hand corner.
+If you are not interested in importing demo data into Atama and your source system, you may skip the quickstart altogether. Simply click the "Quit" button in the upper right hand corner.
+
 
 ### Available starter templates
+![Screenshot of template step](./Select-Template-Quickstart-Atama-Composer.png)
```

Comment:
you know this already, but i'm changing icons for the 'full commerce' demo right now, you'll probably want to retake this screenshot

Reply:
Yeah, this one will probably change some more in the next couple of weeks so I'll leave it like that. The users will get the idea at least.